_posts/blog/2016-09-06-hack-week.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "upday Hack-Week"
+description: Three days, 49 participants, 30+ ideas.
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Three days, 49 participants and 30+ ideas. Find out what we did during our Hack-Week, why we did it, and what we learnt from it.
+tags: [Development, Hackathon, Hack-Week]
+date: 2016-09-06T15:39:55-04:00
+---
+
+After two smaller hackathons; one for Android developers, dedicated to testing and another for the entire engineering, targeting general improvements in automation and productivity, we decided that it's time to go to the next level  and expand the innovation opportunity beyond the engineering teams. So we created "Hack-Week".
+
+Here’s who participated in this, how we pitched the ideas, who worked on implementing them and most of all what we learned from the Hack-Week.
+
+## Who
+
+Developers, product owners, editors, finance, business intelligence or marketing colleagues, all have one thing in common - everyone has lots of ideas! We wanted to give every person a chance to see their ideas come to life. We know that there's always space for improvement on all levels, therefore we opened the Hack-Week to everyone in the company. The Hack-Week gave the opportunity for close collaboration between people who usually have a more distant working relationship. The closeness afforded by the Hack-Week meant the we didn't have those same boundaries.
+
+<center>
+<figure class="third">
```

Comment:
@florina-muntenescu @maciejwalkowiak it seems that after updating the layout there is no more customizable displaying of pictures - so either we need to fix that or you can just add the pictures without non existing css-classes